### PR TITLE
Test user api

### PR DESCRIPTION
**Pull request by dungvan**

### CODE REVIEW

File: tests/Feature/UserApiTest.php [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+                [
+                    'full_name' => 'test register',
+                    'email' => 'test.user@gmail.com',
+                    'password' => '123457',
+                    'password_confirmation' => '123456'
+                ],
+                [
+                    'password' => ['The password confirmation does not match.']
+                ]
+            ]
```
Comment (ghost) [inline tests/Feature/UserApiTest.php:170]: Please add case check exists email in DB

**Review by ghost — COMMENTED** [review]
Please check my comment

File: tests/Feature/UserApiTest.php (hunk 1 above)
Comment (dungvan) [reply tests/Feature/UserApiTest.php:170]: i fixed, check again please! thank you!

**Comment by ghost** [comment]
LGTM

**Pull request merged**